index.js

```diff
@@ -8,17 +8,38 @@ import isUrl from 'is-url'
 import {visitParents} from 'unist-util-visit-parents'
 import {is} from 'unist-util-is'
 
-const isImgExt = (/** @type {string} */ value) =>
-  /\.(svg|png|jpg|jpeg|gif)$/.test(value)
+/**
+ * @typedef Options
+ * @property {Array<string>} [imageExtensions] file extensions of images
+ */
+
 const isImgPath = (/** @type {string} */ value) =>
   value.startsWith('/') || value.startsWith('./') || value.startsWith('../')
 
+/**
+ * Extensions recognized as images by default
+ */
+export const defaultImageExtensions = [
+  'svg',
+  'png',
+  'jpg',
+  'jpeg',
+  'gif',
+  'webp',
+  'avif'
+]
+
 /**
  * Plugin to add an improved image syntax.
  *
- * @type {import('unified').Plugin<void[], Root>}
+ * @type {import('unified').Plugin<[Options?], Root>}
```

Comment:
```suggestion
 * @type {import('unified').Plugin<[Options?]|void[], Root>}
```

I believe there are some cases where `[Options?]` was going wrong in the past. I’m not exactly sure what it was anymore though, maybe that `undefined` wasn’t supported or so

Reply:
added